openmetadata-service/src/main/java/org/openmetadata/service/apps/bundles/insights/DataInsightsApp.java

```diff
@@ -0,0 +1,118 @@
+      // Create Data Insights Pipeline
+      CreateIngestionPipeline createPipelineRequest =
+          new CreateIngestionPipeline()
+              .withName(DEFAULT_INSIGHT_PIPELINE)
+              .withDisplayName(DEFAULT_INSIGHT_PIPELINE)
+              .withDescription("Data Insights Pipeline")
+              .withPipelineType(PipelineType.DATA_INSIGHT)
+              .withSourceConfig(
+                  new SourceConfig()
+                      .withConfig(
+                          new MetadataToElasticSearchPipeline()
+                              .withType(
+                                  MetadataToElasticSearchPipeline.MetadataToESConfigType.METADATA_TO_ELASTIC_SEARCH)))
+              .withAirflowConfig(getDefaultAirflowConfig())
+              .withService(omService.getEntityReference());
+      // Get Pipeline
+      IngestionPipeline dataInsightPipeline =
+          getIngestionPipeline(createPipelineRequest, String.format("%sBot", app.getName()), "admin")
+              .withProvider(ProviderType.SYSTEM);
+      ingestionPipelineRepository.setFullyQualifiedName(dataInsightPipeline);
+      ingestionPipelineRepository.initializeEntity(dataInsightPipeline);
+
+      // Add Ingestion Pipeline to Application
+      collectionDAO
+          .relationshipDAO()
+          .insert(
+              app.getId(),
+              dataInsightPipeline.getId(),
+              Entity.APPLICATION,
+              Entity.INGESTION_PIPELINE,
+              Relationship.CONTAINS.ordinal());
+    } catch (Exception ex) {
+      LOG.error("[IngestionPipelineResource] Failed in Creating Reindex and Insight Pipeline", ex);
+      LOG.error("Failed to initialize DataInsightApp", ex);
+      throw new RuntimeException(ex);
+    }
+  }
+
+  private IngestionPipeline getIngestionPipeline(CreateIngestionPipeline create, String botname, String user) {
+    IngestionPipelineRepository ingestionPipelineRepository =
+        (IngestionPipelineRepository) Entity.getEntityRepository(Entity.INGESTION_PIPELINE);
+    OpenMetadataConnection openMetadataServerConnection =
+        new OpenMetadataConnectionBuilder(ingestionPipelineRepository.getOpenMetadataApplicationConfig(), botname)
+            .build();
+    return ingestionPipelineRepository
+        .copy(new IngestionPipeline(), create, user)
+        .withPipelineType(create.getPipelineType())
+        .withAirflowConfig(create.getAirflowConfig())
+        .withOpenMetadataServerConnection(openMetadataServerConnection)
+        .withSourceConfig(create.getSourceConfig())
+        .withLoggerLevel(create.getLoggerLevel())
+        .withService(create.getService());
+  }
+
+  public static AirflowConfig getDefaultAirflowConfig() {
+    return new AirflowConfig()
+        .withPausePipeline(false)
+        .withConcurrency(1)
+        .withPipelineTimezone("UTC")
+        .withRetries(3)
+        .withRetryDelay(300)
+        .withPipelineCatchup(false)
+        .withScheduleInterval("0 0 * * *")
```

Comment:
why are we hardcoding this?

Reply:
this is default airflow config shipped with the app, like we used to do earlier as well, user can later on update this

Reply:
@mohityadav766 this should come from manifest rather than hard-coding in app

Reply:
fixed